test/e2e/specs/wp-gutenboarding-spec.js

```diff
@@ -149,7 +149,7 @@ describe( 'Gutenboarding: (' + screenSize + ')', function () {
 		} );
 	} );
 
-	describe( 'Visit Gutenboarding page as a logged in user', function () {
+	describe( 'Visit Gutenboarding page as a logged in user @parallel', function () {
```

Comment:
This test wasn't running because it (or its parents) didn't have the `@parallel` tag.